ui/app/mainui/SendModalHandler.qml

```diff
@@ -0,0 +1,152 @@
+import QtQuick 2.15
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Utils 0.1 as SQUtils
+
+import AppLayouts.Wallet.stores 1.0 as WalletStores
+
+import shared.popups.send 1.0
+import shared.stores.send 1.0
+
+import utils 1.0
+
+QtObject {
+    id: root
+
+    required property var popupParent
+    required property int loginType
+    required property TransactionStore transactionStore
+    required property WalletStores.CollectiblesStore walletCollectiblesStore
+
+    // for ens flows
+    required property string myPublicKey
+    required property string ensRegisteredAddress
+    // TODO: This should probably be a property and not a function. Needs changes on  backend side
+    property var getStatusTokenKey: function() {}
+
+    // for sticker flows
+    required property string stickersMarketAddress
+    required property string stickersNetworkId
```

Comment:
How practical is it to make all these props required, while in fact only a subset of them is truly needed in a given scenario/intent?